sklearn/tree/_classes.py

```diff
@@ -932,15 +927,14 @@ def predict_log_proba(self, X):
 
         Parameters
         ----------
-        X : array-like or sparse matrix of shape (n_samples, n_features)
+        X : {array-like, sparse matrix} of shape (n_samples, n_features)
             The input samples. Internally, it will be converted to
             ``dtype=np.float32`` and if a sparse matrix is provided
             to a sparse ``csr_matrix``.
 
         Returns
         -------
-        proba : array of shape (n_samples, n_classes), or a list of n_outputs \
-            such arrays if n_outputs > 1.
+        proba : ndarray of shape (n_samples, n_classes) or list of int
```

Comment:
it's not a list of int it's a list ndarrays where the list is of size n_outputs, I'd keep the original description